packages/dev-env/src/mock/index.ts

```diff
@@ -121,6 +121,65 @@ export async function generateMockSetup(env: DevEnv) {
   await follow(carla, alice)
   await follow(carla, bob)
 
+  // everybody's in the "besties" scene
+  const bestiesScene = await alice.api.app.bsky.actor.createScene({
+    handle: 'besties.test',
+  })
+  {
+    const invite = await alice.api.app.bsky.graph.assertion.create(
+      { did: bestiesScene.data.did },
+      {
+        assertion: APP_BSKY_GRAPH.AssertMember,
+        subject: {
+          did: bob.did,
+          declarationCid: bob.declarationCid,
+        },
+        createdAt: new Date().toISOString(),
+      },
+    )
+    bob.api.app.bsky.graph.confirmation.create(
```

Comment:
I think we want these `await`s, but feel free to ignore if it was intentional as-is.
```suggestion
    await bob.api.app.bsky.graph.confirmation.create(
```